tests/endpoints/bookstore_grpc/proto/v1/bookstore.proto

```diff
@@ -172,6 +165,22 @@ message GetShelfRequest {
   int64 shelf = 1;
 }
 
+message ObjectOnlyForAny {
+  int32 id = 1;
```

Comment:
So that means the open source protocol tool actually compiled this into proto descriptor. 

No need to add it into Types, which is different from internal Blaze rule.

Also, Seems ESPv1 need to do this.

Reply:
Can you explain the reason?

Reply:
I suspected it is by some config option in api_service buildRule.
Raised the question on yaqs/5494949928370176.